tests/testthat/_snaps/content.md

```diff
@@ -0,0 +1,58 @@
+# invalid inputs give useful errors
+
+    Code
```

Comment:
We always use snapshot tests for errors since they also include the call, making it easier to skim for issues.